README.md

```diff
@@ -223,4 +340,43 @@ Response
   "deviceModel": "Samsung", // nullable
   "deviceType": "mobile" // mobile | desktop | web
 }
-```
\ No newline at end of file
+```
+
+## 7. Enums
+
+In this project, enums are used to define the possible values of some fields.
+
+### 7.1 Device Type
+
+Device types are only used for UI purposes like showing an icon.
+
+There is no difference in the protocol between the different device types.
+
+| Value    | Description                               |
+|----------|-------------------------------------------|
+| mobile   | mobile device (Android, iOS, FireOS)      |
+| desktop  | desktop (Windows, MacOS, Linux)           |
+| web      | web browser (Firefox, Chrome)             |
+| headless | program without GUI running on a terminal |
+| server   | (self-hosted) cloud service running 24/7  |
+
+The implementation handle unknown values. The official implementation falls back to `desktop`.
+
+### 7.2 File Type
+
+Different file types may be handled differently by the client.
```

Comment:
i think, use MIME standard[^1][^2] better, no need to implement it yourself

[^1]: https://www.rfc-editor.org/rfc/rfc6838
[^2]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

library:

- https://pub.dev/packages/mime (dart)
- https://pkg.go.dev/net/http#DetectContentType (golang, built-in)

Reply:
Agree, changed to mime.